packages/ui/src/machines/authenticator/actors/signIn.ts

```diff
@@ -549,6 +554,19 @@ export function signInActor({ services }: SignInMachineOptions) {
         shouldSetupTOTP: (_, event): boolean => {
           return isExpectedChallengeName(getChallengeName(event), 'MFA_SETUP');
         },
+        shouldSelectMfaType: (_, event): boolean => {
+          const isSelectedMfaType = isExpectedChallengeName(
```

Comment:
This is confusing, why wouldn't this be false if if it's an unexpected challenge name?

Reply:
Right, it would be false if it's an unexpected challenge name(`SELECT_MFA_TYPE`). a.k.a. It would be false if it's not `SELECT_MFA_TYPE`